.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish packages
+
+on:
+  push:
+    tags: ["*"]
+
+permissions:
+  contents: write
+  id-token: write
+  attestations: write
+
+jobs:
+  github:
+    uses: tree-sitter/workflows/.github/workflows/release.yml@main
```

Comment:
Where exactly is this pointing? Is it this? https://github.com/tree-sitter/tree-sitter/blob/master/.github/workflows/release.yml

I don't really like the idea of pointing to a ci that we have no control over or can change without our input, if that's what this is doing.

Reply:
That's it, it's widely used across "official" parser repos.

It only gets called when a git tag is added, and we have none. I only use that to test wasm compilation on my forks.

Reply:
I still don't like it for the reasons I listed, but I don't have a problem taking your word that it's ok.